SUMMARY.md

```diff
@@ -27,6 +27,13 @@
   * [Encryption](security/encryption.md)
   * [Document storage](security/documents.md)
 
+## Guide
+* [1. Getting an Account](guide/setup-integration-account.md)
```

Comment:
`.gitbook/assets/configure-supplier-account-number.png` may be removed

Reply:
Good one. Removed.